src/components/NotesList/NotesList.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from "prop-types";
+import Box from "@mui/material/Box";
+
+import NoteCard from "./NoteCard/NoteCard";
+
+const NotesList = ({ notes, onNoteSelecting }) => {
+  return (
+    <Box>
+      {notes.map((note, key) => (
+        <NoteCard note={note} key={key} onClick={onNoteSelecting(note)} />
```

Comment:
Why it's not a good practice to use an index as a key for mapped element?
Replace it please with a uniq key (Example: id of note)